ThunderCloud/QuizQuestionContainerViewController.swift

```diff
@@ -422,7 +445,36 @@ extension QuizQuestionContainerViewController: UIGestureRecognizerDelegate {
 extension QuizQuestionContainerViewController: QuizQuestionViewControllerDelegate {
     
     func quizQuestionViewController(_ questionViewController: QuizQuestionViewController, didChangeAnswerFor question: QuizQuestion) {
+        
         redrawSelectedLabel()
         redrawContinueButton()
+        
+        var selected: String?
```

Comment:
Why `String`?

Reply:
No reason really, will change it over to int if we wish :)